questions/28.t-hello.ts

```diff
@@ -0,0 +1,3 @@
+export function hello(name: string): string {
+    return `hellow ${name}`;    
```

Comment:
```suggestion
    return `hello ${name}`;
```

Oh was the `w` intentional?

Reply:
I could say it's intended. 😆. But it really is a typo. 😝

Reply:
hahahaha!